apps/omg_watcher_rpc/test/omg_watcher_rpc/web/controllers/alarm_test.exs

```diff
@@ -0,0 +1,55 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherRPC.Web.Controller.AlarmTest do
+  use ExUnitFixtures
+  use ExUnit.Case, async: false
+
+  use OMG.Fixtures
+  use OMG.Watcher.Fixtures
+  alias OMG.Watcher.TestHelper
+
+  setup %{} do
+    system_alarm = {:system_memory_high_watermark, []}
```

Comment:
Since we have the same setup/tests already at the unit level in `API.AlarmTest`, what do you think about removing them here? We could still leave a test in here of course to make sure thing are working, but maybe just a single/simple test with one alert. What are your thoughts?

Reply:
This is somehow ensuring that `OMG.WatcherRPC.Web.View.Alarm` and `OMG.Utils.HttpRPC.Response` can encode the alarms and properly return them as JSONs. I guess what I'm trying to say is that if I remove them from here, I would prefer to "move" them to `OMG.Utils.HttpRPC.ResponseTest` and check if they're being handled properly.

Reply:
Ahhh, I see. Yeah that makes sense. I think moving to `HttpRPC.ResponseTest` sounds like a pretty good idea. It looks like we have some sanitation tests in there already and this would go well in there.